clients/spark/core/src/main/scala/io/treeverse/clients/GarbageCollector.scala

```diff
@@ -643,7 +648,16 @@ object GarbageCollector {
       .withColumn(MARK_ID_KEY, lit(markID))
   }
 
+  private def readRunIDFromMarkIDMetadata(addressesLocation: String, markID: String): String = {
+    spark.read
+      .json(s"$addressesLocation/$markID.meta")
```

Comment:
Perhaps make it clearer that we interpolate `markID` and not a field of it?

```suggestion
      .json(s"$addressesLocation/${markID}.meta")
```

Reply:
what's the difference (other than IntelliJ yelling at me to remove those curly brackets)?